specifyweb/specify/management/commands/create_geo_trees.py

```diff
@@ -0,0 +1,26 @@
+from typing import Any, Optional
+from django.core.management.base import BaseCommand
+from specifyweb.specify.tree_views import TAXON_TREES, TAXON_RANKS
+from specifyweb.specify.models import Taxontreedef, Taxontreedefitem
+
+class Command(BaseCommand):
+    def handle(self, *args: Any, **options: Any) -> Optional[str]:
+        help = "Add initial geology taxon tree definitions to the database."
+
+        for tree in TAXON_TREES:
+            if Taxontreedef.objects.filter(name=tree).exists():
+                continue
+            tree_name = f"{tree} Taxon"
+            ttd = Taxontreedef.objects.create(name=tree_name)
+            rank_id = -10
+            ttdi = None
+            for rank in TAXON_RANKS:
+                name = f"{tree} {rank}"
+                rank_id += 10
+                ttdi = Taxontreedefitem.objects.create(
+                    name=name,
+                    title=name,
+                    rankid=rank_id,
+                    parent=ttdi,
+                    treedef=ttd,
+                )
```

Comment:
We have this excact same logic elsewhere: https://github.com/specify/specify7/blob/7f5cf14ddb3c7bb4b9077234adf04f7b3dbed0d6/specifyweb/specify/tree_views.py#L379-L396

Can you extract that into a separate function and use it for the view and for this endpoint (or just call that function here if it works)? 
That way, if we want to modify the default Geo Trees, we only need to modify one place.

Reply:
Furthermore, why is there both an endpoint and this command? 
Where are we using this command where the endpoint can't suffice (or vice versa)?